src/index.html

```diff
@@ -14,24 +14,26 @@ <h1>Leaderboard</h1>
     </header>
     <div class="container">
       <div class="recent-score">
-        <h3>Recent scores <button>Refresh</button></h3>
+        <h3>Recent scores <button id="refresh">Refresh</button></h3>
         <div class="score-display">
-       <span>Name:100</span>
+       <!-- <span>Name:100</span>
        <span>Name:20</span>
        <span>Name:50</span>
        <span>Name:78</span>
        <span>Name:125</span>
        <span>Name:77</span>
-       <span>Name:42</span>
+       <span>Name:42</span> -->
```

Comment:
- [ ] **Required:** Kindly remove the unused commented code to make your work clean, more readable and professional.